notebook/analysis/benchmark_results.py

```diff
@@ -581,11 +588,107 @@ def return_unlabeled():
 plot += geom_smooth()
 plot += geom_point(alpha=.2)
 plot
+
+# %% [markdown]
+# ## Results from Small Datasets
+
+# %%
+in_files = glob.glob('../../results/small_subsets.sepsis*')
+print(in_files[:5])
+
+# %%
+sepsis_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('sepsis.')[-1]
+    model_info = model_info.split('.')
+        
+    supervised_model = model_info[0]
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-2]
+        
+    sepsis_metrics = pd.concat([sepsis_metrics, new_df])
+    
+sepsis_metrics['train_count'] = sepsis_metrics['train sample count']
+
+# Looking at the training curves, deep_net isn't actually training
+# I need to fix it going forward, but for now I can clean up the visualizations by removing it
+sepsis_metrics = sepsis_metrics[~(sepsis_metrics['supervised'] == 'deep_net')]
+sepsis_metrics
+
+# %%
+plot = ggplot(sepsis_metrics, aes(x='factor(train_count)', y='balanced_accuracy')) 
+plot += geom_boxplot()
+plot += ggtitle('Sepsis Dataset Size Effects (equal label counts)')
+print(plot)
+
+# %%
+plot = ggplot(sepsis_metrics, aes(x='factor(train_count)', y='balanced_accuracy', fill='supervised')) 
+plot += geom_boxplot()
+plot += ggtitle('Sepsis Datset Size by Model (equal label counts)')
+print(plot)
+
+# %%
+plot = ggplot(sepsis_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
+plot += geom_smooth()
+plot += geom_point(alpha=.2)
+plot += ggtitle('Sepsis Crossover Point')
+plot
+
+# %% [markdown]
+# ## Small Training Set TB
+
+# %%
+in_files = glob.glob('../../results/small_subsets.tb*')
+print(in_files[:5])
+
+# %%
+tb_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('tb.')[-1]
+    model_info = model_info.split('.')
+        
+    supervised_model = model_info[0]
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-2]
+        
+    tb_metrics = pd.concat([tb_metrics, new_df])
+    
+tb_metrics['train_count'] = tb_metrics['train sample count']
+
+# Looking at the training curves, deep_net isn't actually training
+# I need to fix it going forward, but for now I can clean up the visualizations by removing it
+tb_metrics = tb_metrics[~(tb_metrics['supervised'] == 'deep_net')]
+tb_metrics
+
+# %%
+plot = ggplot(tb_metrics, aes(x='factor(train_count)', y='balanced_accuracy')) 
+plot += geom_boxplot()
+plot += ggtitle('TB Dataset Size Effects (equal label counts)')
+print(plot)
+
+# %%
+plot = ggplot(tb_metrics, aes(x='factor(train_count)', y='balanced_accuracy', fill='supervised')) 
+plot += geom_boxplot()
+plot += ggtitle('TB Dataset Size vs Models (equal label counts)')
+print(plot)
+
+# %%
+plot = ggplot(tb_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
```

Comment:
You could try plotting this with error bars (e.g. standard error) to emphasize that the crossover isn't really significant, if you want. (At least I assume it won't be, looking at the data points)

Reply:
I think `geom_smooth` is supposed to plot confidence intervals by default but didn't here for some reason. Maybe the fact that the x axis isn't really continuous?